src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -17,18 +17,23 @@ import { If } from 'react-if';
 import {SortMetric} from "../../lib/ISortMetric";
 import {IMobXApplicationDataStore, SimpleMobXApplicationDataStore} from "../../lib/IMobXApplicationDataStore";
 
+import {shallow, mount, ReactWrapper} from 'enzyme';
+
 export type SortDirection = 'asc' | 'desc';
 
+export type TChild = {};
+
 export type Column<T> = {
     name: string;
     filter?:(data:T, filterString:string, filterStringUpper?:string, filterStringLower?:string)=>boolean;
     visible?:boolean;
     sortBy?:((data:T)=>(number|null)) | ((data:T)=>(string|null)) | ((data:T)=>(number|null)[]) | ((data:T)=>(string|null)[]);
     render:(data:T)=>JSX.Element;
-    download?:(data:T)=>string;
+    download?:(data:T, sample?:TChild)=>string;
     tooltip?:JSX.Element;
     defaultSortDirection?:SortDirection;
     togglable?:boolean;
+    toDownload?:boolean;
```

Comment:
I don't think we need toDownload, instead we can check if download is defined or not